articles/110_interface_definition.md

```diff
@@ -104,8 +104,19 @@ If no default value is specified a common default value is used:
 - for *numeric* types it is a `0` value
 - for `string` it is an *empty* string
 
+#### Array default values
+
+A field of type `array` can optionally specify a default value.
+
+- default values for an array must start with an opening square bracket (`[`) and end with a closing square bracket (`]`)
+- each value within the array must be separated with a comma (`,`)
```

Comment:
Also, it's obvious to me, but it might be worth mentioning that the values in the array must all be of the same type, i.e. the array is homogeneous.